agents/wiregrid_kikusui/src/pmx.py

```diff
@@ -0,0 +1,341 @@
+# Built-in python modules
+import time as tm
+import serial as sr
+import sys as sy
+import os
+
+# Communication modules
+this_dir = os.path.dirname(__file__)
+sy.path.append(os.path.join(
+    this_dir, "..", "..","..", "MOXA"))
+import moxaSerial as mx  # noqa: E402
+
+class PMX:
+    """
+    The PMX object is for communicating with the Kikusui PMX power supplies
+
+    Args:
+    rtu_port (str): Serial RTU port
+    tcp_ip (str): TCP IP address
+    tcp_port (int): TCP port
+    """
+    def __init__(self, rtu_port=None, tcp_ip=None, tcp_port=None, timeout=None):
+        # Connect to device
+        msg = self.__conn(rtu_port, tcp_ip, tcp_port, timeout)
+        print(msg)
+        self._remote_Mode()
+
+        # Timing variables
+        self._tstep = 0.1  # sec
+
+    def __del__(self):
+        if not self.using_tcp:
+            print(
+                "Disconnecting from RTU port %s"
+                % (self._rtu_port))
+            self.ser.close()
+        else:
+            print(
+                "Disconnecting from TCP IP %s at port %d"
+                % (self._tcp_ip, self._tcp_port))
+            pass
```

Comment:
Unnecessary `pass`. All other `pass` statements in this file can also be removed.